src/core/thread/mle_router_ftd.hpp

```diff
@@ -65,6 +65,18 @@ class NetworkDataLeader;
  */
 
 /**
+* This structure represents the child information for persistent storage.
+*
+*/
+struct ChildInfo
+{
+    Mac::ExtAddress  mExtAddress;    ///< Extended Address
+    uint32_t         mTimeout;       ///< Timeout
+    uint16_t         mRloc16;        ///< RLOC16
+    uint8_t          mMode : 4;      ///< The MLE device mode
```

Comment:
Yes, updated. I also updated the mMode definition in **class Neighbor** as well.